src/expressions/visitors/EvalVisitor.cpp

```diff
@@ -130,6 +131,17 @@ EvaluationResult EvalVisitor::visit(const Nodes::ComponentParameterNode* node)
     throw EvalVisitorNotImplemented(name(), node->name());
 }
 
+EvaluationResult EvalVisitor::visit(const Nodes::TimeShiftNode* node)
+{
+    const auto ret = dispatch(node->child());
```

Comment:
I would aprpeciate to know the type of `ret` here; rather than `auto`

Reply:
it is EvaluationResult ;)